pkg/querier/cardinality_analysis_handler_test.go

```diff
@@ -705,10 +705,6 @@ func TestLabelValuesCardinalityHandler_ParseError(t *testing.T) {
 				url:                  "/label_values",
 				expectedErrorMessage: "'label_names[]' param is required",
 			},
-			"label_names param is invalid": {
-				url:                  "/label_values?label_names[]=olá",
-				expectedErrorMessage: "invalid 'label_names' param 'olá'",
-			},
```

Comment:
The test case just needs an adjustment, to a string which isn't valid UTF-8:
```suggestion
			"label_names param is invalid": {
				url:                  "/label_values?label_names[]=\xff\xfe",
				expectedErrorMessage: "invalid 'label_names' param '\xff\xfe'",
			},
```